external-table/src/pxfheaders.c

```diff
@@ -90,6 +92,20 @@ build_http_headers(PxfInputData *input)
 		ListCell   *option;
 		List	   *copyFmtOpts = NIL;
 
+		// in the case of pxfdelimited_import formatter, the only viable profiles are TEXT and CSV.
+		// error out early here if the profile is not accepted
+		if (getFormatterString(exttbl) != NULL &&
```

Comment:
```suggestion
		if (getFormatterString(exttbl) &&
```
do we need explicit NULL comparisons in C since a pointer will be >0 and evaluate to `true` in conditionals ?